traits/tests/test_property_notifications.py

```diff
@@ -26,52 +27,19 @@ def __value_set(self, value):
             self._value = value
             self.trait_property_changed("value", old_value, value)
 
-    __traits__["value"] = Property(__value_get, __value_set)
+    value = Property(__value_get, __value_set)
```

Comment:
@mdickinson Making this change got the prints working. The notification does not get triggered if the trait is added to the dictionary explicitly. Is this a bug ?

Reply:
Honestly, I have no idea why the test was written the way it was. It does make sense that adding to the `__traits__` dictionary directly wouldn't work, so no, not a bug in Traits itself.

Reply:
And to be clear, the reason that adding to `__traits__` has no effect is that `__traits__` is not special in any way. This is the only file in the codebase that even mentions it.

Reply:
Got it. Thanks!